tutorial/about-tutorial.md

```diff
@@ -1,101 +1,58 @@
 
-### The Before you Start section
+If you're writing a tutorial where you provide users with a starter project, you could describe the project hierarchy here. For example, a tutorial around building a website with HTML, CSS, and JavaScript might explain how the folders are organized, what code is provided for the user, or what they need to add.
 
-This section prevents readers from getting halfway through and discovering that they need to go and read other documentation before they can continue.
-Prerequisites can include other articles or information to read, or it can be technical dependencies such as requiring an internet connection, or particular software.
+### About the "Before you begin" section
 
-Describe what the audience needs to know, or needs to have, before they attempt the How To.
-By stating the requirements up-front, you prevent your readers from having a bad experience with your How To.
-You can include links to procedures or information, or give useful pointers about how to get what they need.
+Use this section to list any prerequisites for completing the tutorial. This may include operating systems, languages, package managers, or software.
 
-For example:
+### About the "Steps" section
 
-```markdown
-Before you begin, make sure you meet these prerequisites:
+The steps section is where you describe what the user needs to do. How you write your steps will vary depending on your organization's style guide. 
 
-* API credentials for the v3.5 API. For more information about accessing your API credentials, see http://example.com/access_your_api_credentials.
-* Access to the Postman application.
-* A conceptual understanding about RESTful APIs. For more information, see http://example.com/restful_apis.
-* (Optional) A development environment (IDE) that displays API responses formatted for readability.
-* A list of favorites prepared, so you can manage them. For more information about favorites lists, see http://example.com/favorite_lists.
+This template breaks down the tutorial into _parts_. One _part_ of the tutorial may focus on completing several related steps. You're welcome to follow this structure or use the step headings on their own.
 
-```
+#### Tips for writing steps in a tutorial
 
-### The Part sections
+* When writing your steps, make sure to provide any background information your users may need to complete the task. 
+* Remember to orient your users when walking them through each step. If they need to open a particular file or dialog to complete the task, provide that information first.
 
-Use either a structured style, or an informal style in this section.
+### About the "Summary" section
 
-#### Structured style
+Use the summary section of the tutorial to sum up what tasks your user completed during the tutorial and what your users now know how to do.
```

Comment:
**Suggestion (non-blocking, not in response to Cameron's edit but is commenting on the same section of text):** I feel like I want some more meatier advice about the Summary section. Maybe talk more about the purpose of this section and how it helps the reader. In my mind, the summary section can accomplish many possible rhetorical tasks. Yes, it can summarize what concepts they learned, but it can also talk about why those concepts matter and how they connect to a larger, deeper workflow. It can also have a small marketing function: look at how cool this tool is and how easy the features are to use!

The key is to ask what do you want your reader to think, feel, or do after they've completed the tutorial. What do you want them to notice, what do you want them to be cautious about, etc.